src/main/resources/errors.yaml

```diff
@@ -202,6 +202,20 @@ request-errors:
       
       ${SNIPPET.INEFFICIENT_QUERY}
 
+
+  - scope: WARNING
+    code: COMPARISON_FILTER_UNSUPPORTED_BY_INDEXING
+    title: Use of $lt/$gt/$lte/$gte (comparison filter) on indexed columns
+    body: |-
+      The filter uses $lt/$gt/$lte/$gte (comparison filters) on columns that, while indexed, are still inefficient to filter on.  
+      
+      Filtering using $lt/$gt/$lte/$gte on columns of type ${inefficientDataTypes} is inefficient, even when the columns are indexed.
+      
+      The table ${keyspace}.${table} uses these data types for the columns: ${allUuidColumns}.
```

Comment:
nit: use the `inefficientColumns` name for the var

Reply:
fixed, thanks